### PR TITLE
dbtime with dba_hist_*

### PR DESCRIPTION
**Pull request by yangdsh**
There are some differences between the dba_hist view and the system_event view. For example, there is no average_wait_time in dba_hist view, so I use wait_time/total_waits to calculate the default average wait time. Also, the wait_class_ids of the dba_hist_view are all very large int numbers, so I need to use wait_class name instead.

### CODE REVIEW

File: server/website/website/db/base/parser.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ def convert_dbms_metrics(self, metrics, observation_time, target_objective):
 
         for target_name, target_instance in target_list.items():
             metric_data[target_name] = target_instance.compute(
-                base_metric_data, observation_time)
+                metrics, observation_time)
```
Comment (yangdsh) [inline server/website/website/db/base/parser.py:230]: In order to use the wait class name in target objectives, I use the metrics before conversion to calculate the target objectives.

**Comment by yangdsh** [comment]
This PR lacks the fixture file. I do not know which metrics to include in the fixture files. We might want to only use dba_hist_* views, because other views are all duplicated. There are too many metrics now, which makes the processing especially metric pruning slow.

**Comment by yangdsh** [comment]
When using dba_hist_* to calculate db time, the raw db time from the dba_hist_sys_time_model is exactly the same as that in the awr. However, when we sum up the wait time, the result is still a little larger (5-10%) than the db time in the awr.

**Comment by dvanaken** [comment]
The main problem I see with only including metrics from the dba_hist_* views is that we won't be able to upload our past results from SocGen to newer versions of the webserver.

**Pull request merged**